commerce-service/src/main/java/com/liferay/commerce/service/impl/CommerceOrderServiceImpl.java

```diff
@@ -458,8 +458,8 @@ public int getPlacedCommerceOrdersCount(
 
 		return commerceOrderLocalService.getCommerceOrders(
 			companyId, groupId, commerceAccountIds, keywords,
-			new int[] {CommerceOrderConstants.ORDER_STATUS_OPEN}, false, start,
```

Comment:
@riccardo-alberti this is not a correct fix.  the method is getPlacedOrders . and was getting all orders that are not Open, why the change?

Reply:
@marco-leo i changed the method getUserPendingCommerceOrders so it uses the orderstastus that's already used in the admin section when listing the pending orders.

Reply:
@riccardo-ferrari Admin UI and Front portlet have different logic and meanings, is correct that they show different data.